src/main/java/com/example/demo/model/Book.java

```diff
@@ -25,5 +29,8 @@ public class Book {
     @Column(nullable = false)
     private BigDecimal price;
     private String description;
+    @Column(name = "cover_image")
```

Comment:
this changes nothing. spring will automatically map your entity field name `coverImage` into sql `COVER_IMAGE`
```suggestion
```